pkg/restapi/oidc/operations_test.go

```diff
@@ -502,6 +503,27 @@ func TestOperation_OIDCCallbackHandler(t *testing.T) {
 		require.Equal(t, http.StatusInternalServerError, w.Code)
 		require.Contains(t, w.Body.String(), "create operational keystore")
 	})
+
+	t.Run("failure to create used sds vault", func(t *testing.T) {
```

Comment:
Typo here - should be `user` instead of `used`

Reply:
[done](https://github.com/trustbloc/edge-agent/compare/f72677242d790bd9c9ea7e6785e9df7517f0ac84..e93ac5731b540cd1c7af5454293fd44f23408e1e)